content/patterns/radio/examples/radio.html

```diff
@@ -163,25 +159,25 @@ <h2 id="rps_label">Role, Property, State, and Tabindex Attributes</h2>
               <td><code>div</code></td>
               <td>
                 <ul>
-                  <li>Identifies the <code>div</code> element as a container for a group of <code>radio</code> buttons.</li>
-                  <li>Is not focusable because focus is managed using a roving tabindex strategy as described below.</li>
+                  <li><code>div</code> 要素を <code>radio</code> ボタンのグループのコンテナとして特定。</li>
```

Comment:
「特定」より「明示」くらいのほうがニュアンスに沿うかなあと思いました

```suggestion
                  <li><code>div</code> 要素を <code>radio</code> ボタンのグループのコンテナとして明示する。</li>
```